Jenkinsfile

```
@@ -37,6 +37,11 @@ pipeline{
                     }
 
                     githubscm.checkoutIfExists('kogito-images', changeAuthor, changeBranch, 'kiegroup', changeTarget, true)
+
+                    //Ignore self-signed certificates if MAVEN_MIRROR_URL is defined
+                    if(env.MAVEN_MIRROR_URL != ''){
+                        sh "python3 scripts/update-tests.py --ignore-self-signed-cert"
```

as long as you are making usage of gstring (which means include a variable of a string with `${VAR}`), you should use single-quote ;)
```suggestion
                        sh 'python3 scripts/update-tests.py --ignore-self-signed-cert'
```

---

Ah okay, thanks, will keep in mind from next time :)